services/docker-compose.yml

```
@@ -115,10 +115,31 @@ services:
+      kafka-broker:
+        condition: service_healthy
+      clickhouse:
+        condition: service_started
+    ports:
+      - "3000:3000"
+      - "5000:5000"
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    networks:
+      - deployment
+    restart: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)